src/Swarm/TUI/Controller.hs

```diff
@@ -145,7 +145,7 @@ handleNewGameMenuEvent scenarioStack@(curMenu :| rest) s = \case
             & gameState .~ gs'
       Just (SICollection _ c) ->
         continue $
-          s & uiState . uiMenu .~ NewGameMenu (NE.cons (mkScenarioList c) scenarioStack)
+          s & uiState . uiMenu .~ NewGameMenu (NE.cons (mkScenarioList (s ^. gameState . creativeMode) c) scenarioStack)
```

Comment:
I don't see how the `creativeMode` flag in the `gameState` is relevant here.  It doesn't really tell us much unless we're currently playing a game.  In particular it does *not* tell us whether Swarm was started with the `--cheat` flag enabled.

Reply:
Well spotted! :+1: Weird that I did not notice, but I guess I did not test it too much. :sweat_smile:

Reply:
Now it is `(s ^. uiState . uiCheatMode)` as it should have been. :heavy_check_mark: